Packs/AggregatedScripts/ReleaseNotes/1_0_8.md

```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### get-endpoint-data
+
+Improved implementation of endpoint data retrieval by standardizing argument naming from agent-based to endpoint-based parameters, enhancing data structure consistency, and adding support for additional fields.
```

Comment:
I think it's better splitting this into several points each highlight explicily a different modification.